src/main/java/org/entur/gbfs/loader/BaseGbfsLoader.java

```diff
@@ -56,8 +56,13 @@ public synchronized void init() {
 
     if (discoveryFileUpdater.fetchOnce()) {
       disoveryFileData = discoveryFileUpdater.getData();
-      createUpdaters();
-      setupComplete.set(true);
+
+      try {
+        createUpdaters();
```

Comment:
Shouldn't discoveryFileUpdate.fetchOnce() be included in the try statement? Otherwise e.g. a failing authentication would have the same result, no?

Reply:
Yes!

Reply:
Addressed in [55d0bc8](https://github.com/entur/gbfs-loader-java/pull/168/commits/55d0bc8ed181422348f9a8a852543459e73dfc92)